lispusers/JSON

```diff
@@ -180,11 +177,15 @@
          NIL])
 
 (JSON-GET
-  [LAMBDA (OBJECT ATTRIBUTE)                                 (* ; "Edited 30-Apr-2024 14:26 by rmk")
+  [LAMBDA (OBJECT ATTRIBUTES)                                (* ; "Edited 13-May-2024 19:21 by rmk")
+                                                             (* ; "Edited 30-Apr-2024 14:26 by rmk")
 
-    (* ;; "Returns the value of ATTRIBUTE in OBJECT")
+    (* ;; "Returns the value at the end of a chain of ATTRIBUTES in OBJECT")
 
-    (CADR (ASSOC ATTRIBUTE OBJECT])
+    (for A (OBJ _ OBJECT) inside (OR ATTRIBUTES (REVERSE ATTRIBUTES))
```

Comment:
I'm not sure what effect `(OR ATTRIBUTES (REVERSE ATTRIBUTES))` was intending to produce... if ATTRIBUTES is non-NIL that's the result, otherwise (REVERSE NIL) is NIL ?